Jenkinsfile

```diff
@@ -28,6 +29,19 @@ elifePipeline {
                 ]
             ])
         }
+
+        stage 'Generate critical CSS', {
+            try {
+                sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml run --name=journal_critical_css critical_css"
+                sh "docker cp journal_critical_css:build/critical-css/. build/critical-css/"
+            } finally {
+                sh "docker-compose -f docker-compose.yml -f docker-compose.ci.yml rm -v --stop --force"
+            }
+        }
+
+        stage 'Rebuild FPM image', {
+            sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml build fpm"
+        }
```

Comment:
The stages check out, I'll try to follow up later with separating the pre- and post- `fpm` images.